AirCasting/SessionViews/MeasurementsView.swift

```diff
@@ -0,0 +1,104 @@
+// Created by Lunar on 07/06/2021.
+//
+
+import SwiftUI
+
+struct ABMeasurementsView: View {
+    
+    @ObservedObject var session: SessionEntity
+    var threshold: SensorThreshold
+    @Binding var selectedStream: MeasurementStreamEntity?
+    
+    private var streamsToShow: [MeasurementStreamEntity] {
+        let allStreams = [session.pm1Stream,
+                          session.pm2Stream,
+                          session.pm10Stream,
+                          session.FStream,
+                          session.HStream]
+        
+        let toShow = allStreams.compactMap { $0 }
+        return toShow
+    }
+    private var hasAnyMeasurements: Bool {
+        streamsToShow.filter { $0.latestValue != nil }.count > 0
```

Comment:
I think [contains(where:)](https://developer.apple.com/documentation/swift/array/2297359-contains) would be a better fit here